recipe/bld.bat

```diff
@@ -1,14 +1,2 @@
 set PYTHONHASHSEED=0
-%PYTHON% -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
-if errorlevel 1 exit 1
-
-set ACTIVATE_DIR=%PREFIX%\etc\conda\activate.d
-set DEACTIVATE_DIR=%PREFIX%\etc\conda\deactivate.d
-mkdir %ACTIVATE_DIR%
-mkdir %DEACTIVATE_DIR%
-
-copy %RECIPE_DIR%\scripts\activate.bat %ACTIVATE_DIR%\python-libarchive-c-activate.bat
-if errorlevel 1 exit 1
-
-copy %RECIPE_DIR%\scripts\deactivate.bat %DEACTIVATE_DIR%\python-libarchive-c-deactivate.bat
-if errorlevel 1 exit 1
+python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I'm confused. The issue was pip or the activation?
I could not test it on OS X locally but Linux and Windows were passing and Travis-CI was OK :-/

Reply:
Answering my own questions, it looks like we don't have `${SHLIB_EXT}` defined outside of the build scope :-(

xref: https://github.com/conda-forge/python-libarchive-c-feedstock/issues/6#issuecomment-428023824

Reply:
Oh I find PIP a bit of a pain, most recently not creating entry point scripts on Windows. Feel free to revert this part if pip is the official policy (though I'd propose a policy where we use the simplest tool that's capable of building a correct package in each case).